File: javascript/fizzbuzz.js
function fizzbuzz(num) {
  for(var x = 1; x <= num; x++) {
    if(x % 15 === 0) {
      console.log("FizzBuzz")
    } else if (x % 3 === 0) {
      console.log("Fizz")
    } else if (x % 5 === 0) {
      console.log("Buzz")
    } else {
      console.log(x)
    }
  }
}

fizzbuzz(100)
console.log()
fizzbuzz(50)

File: ruby/fizzbuzz.rb
def fizz_buzz(num)

  range = (1..num).to_a

  range.each do |n|
    if n % 15 == 0
      puts "FizzBuzz"
    elsif n % 3 == 0
      puts "Fizz"
    elsif n % 5 == 0
      puts "Buzz"
    else
      puts "#{n}"
    end
  end
end

fizz_buzz(100)
puts
fizz_buzz(50)
